File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio7 media de tres notas/archives/7.php
<?php
$num1 = ($_POST["num1"]);
$num2 = ($_POST["num2"]);
$num3 = ($_POST["num3"]);
if (is_numeric($num1) && is_numeric($num2) && is_numeric($num3)) {
	$num1 = intval($num1);
	$num2 = intval($num2);
	$num3 = intval($num3);
	if ($num1 >= 0 && $num2 >= 0 && $num3 >= 0) {
		$promedio = "El promedio de las tres notas es: " . round((($num1 + $num2 + $num3) / 3), 2);
	} else {
		$promedio = " Uno varios datos no son validos";
	}
} else {
	$promedio = "Ingrese datos en el formulario";
}
print $promedio;

File: Tema 3 -Sentencia condicional (if-switch)/ejercicios/ejercicio7 media de tres notas/archives/7.py
import sys
import math

# Obtener los argumentos de la línea de comandos
num1 = int(sys.argv[1])
num2 = int(sys.argv[2])
num3 = int(sys.argv[3])
if num1 >= 0 and num2 >= 0 and num3 >= 0:
    mensaje = f"Su promedio es de {round(((num1+num2+num3)/3),2)}"
else:
    mensaje = f"Alguno de los datos no son validos"

print(f"{mensaje}")
